content/authors/Meenakshi-Siddharthan/_index.md

```diff
@@ -15,8 +15,11 @@ education:
 
 
 email: "m.siddharthan@ntu.edu.sg"
-#interests: Lifelong learning, Cognition, Brain stimulation
-
+interests: 
+-Lifelong learning
+-Cognition
+-Brain stimulation
+#is this right now?
```

Comment:
For this to be a list you have to make sure there's a space between the bullet point and the word :) so like:
- Lifelong learning
and not
-Lifelong learning

:smile: